packages/core/src/queries/log.ts

```diff
@@ -44,3 +44,24 @@ export const findLogs = async (limit: number, offset: number, logCondition: LogC
     limit ${limit}
     offset ${offset}
   `);
+
+const registerLogTypes: LogType[] = [
+  'RegisterUsernamePassword',
+  'RegisterEmail',
+  'RegisterSms',
+  'RegisterSocial',
+];
+
+export const getDailyNewUserCountsByTimeInterval = async (
+  startTimeInclusive: number,
+  endTimeExclusive: number
+) =>
+  envSet.pool.any<{ date: string; count: number }>(sql`
+    select date(${fields.createdAt}), count(*)
+    from ${table}
+    where ${fields.createdAt} >= to_timestamp(${startTimeInclusive}::double precision / 1000)
+    and ${fields.createdAt} < to_timestamp(${endTimeExclusive}::double precision / 1000)
```

Comment:
Shouldn't it be `<=`?

Reply:
refactored according to the sijie's comments. PTAL again